docs/migrations.md

```diff
@@ -0,0 +1,69 @@
+# v1alpha3 HAProxy to v1alpha3 kube-vip
+
+to migrate from an existing `HAProxyLoadBalancer` to `kube-vip` you will need the following steps:
+
+1 - get the HAProxyLoadBalancer existing IP with `kubectl get haproxyloadbalancer CLUSTER_NAME -n NAMESPACE -o template='{{.status.address}}'`
+2 - in following patch `patch.yaml` file, replace the `vip_address` value with the IP of the `HAProxyLoadBalancer`:
+
+```yaml
+spec:
+  kubeadmConfigSpec:
+    files:
+    - content: |
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          creationTimestamp: null
+          name: kube-vip
+          namespace: kube-system
+        spec:
+          containers:
+          - args:
+            - start
+            env:
+            - name: vip_arp
+              value: "true"
+            - name: vip_leaderelection
+              value: "true"
+            - name: vip_address
+              value: <HAPROXY_IP>
+            - name: vip_interface
+              value: eth0
+            - name: vip_leaseduration
+              value: "15"
+            - name: vip_renewdeadline
+              value: "10"
+            - name: vip_retryperiod
+              value: "2"
+            image: plndr/kube-vip:0.2.0
+            imagePullPolicy: IfNotPresent
+            name: kube-vip
+            resources: {}
+            securityContext:
+              capabilities:
+                add:
+                - NET_ADMIN
+                - SYS_TIME
+            volumeMounts:
+            - mountPath: /etc/kubernetes/admin.conf
+              name: kubeconfig
+          hostNetwork: true
+          volumes:
+          - hostPath:
+              path: /etc/kubernetes/admin.conf
+              type: FileOrCreate
+            name: kubeconfig
+        status: {}
+      owner: root:root
+      path: /etc/kubernetes/manifests/kube-vip.yaml
+```
+
+3 - patch `KubeadmControlPlane` with the following command `kubectl patch kcp CLUSTER_NAME -n NAMESPACE --type merge  --patch "$(cat patch.yaml)"` replace `KCP_NAME` and `NAMESPACE` with the `KubeadmControlPlane` name and namespace
```

Comment:
might be worth to add a note to check that this patch does not override custom changes to the default templates

Reply:
Also in the command it has CLUSTER_NAME and later it says to replace KCP_NAME